### DISCUSSION TITLE
user name case-sensitive

### DISCUSSION THREAD

**Discussion by henrywithu** [requarks/wiki#3588] [Error / Bug Report]
when login, user name is case-sensitive, which is not quite friendly to mobile users.
cuz when u input on mac, it's most likely lower case; on phone/pad, auto upper case

**Comment by 3DSkaoi** [comment]
As you may have noticed, the language used for WIKi.JS is javascript. You may not know, but javascript is case sensitive, so is Linux username. 

You can always submit a feature request regarding this matter.
